meinberlin/assets/js/i18n-leaflet-draw.js

```diff
@@ -0,0 +1,106 @@
+window.L.drawLocal = {
+  draw: {
+    toolbar: {
+      actions: {
+        title: 'Bereich entfernen',
+        text: 'abbrechen'
```

Comment:
Some texts are lowercase, most uppercase. I would suggest to have everything uppercase.